spec/javascript/support/puppeteer/steps/leaseUpStatusSteps.js

```diff
@@ -0,0 +1,59 @@
+import sharedSteps from './sharedSteps'
+
+export const openStatusModalSelector = '.form-modal_form_wrapper'
+
+export const commentInputSelector = '#status-comment'
+
+export const submitStatusModalSelector = '.form-modal_form_wrapper button.primary'
+
+export const unselectedStatusMenuItem = 'li[aria-selected="false"].dropdown-menu_item > a'
+export const selectStatusDropdownValue = async (page, dropdownSelector, valueSelector) => {
+  await page.click(dropdownSelector)
+  await page.waitForSelector('li.dropdown-menu_item')
+  await page.click(valueSelector)
+  await page.waitForSelector(openStatusModalSelector)
+}
+
+export const selectSubstatusIfRequired = async (selectedStatus, page) => {
+  if (
+    selectedStatus.toLowerCase() !== 'processing' &&
```

Comment:
instead of this are you able to just do a check if the substatus dropdown is present? I could see the business logic for this changing, like we add a new status without a substatus and we wouldn't remember to change this code

Reply:
Oh I see now that this code was just ported over, it's not new code. We can leave it.